### PR TITLE
More copyright

### PR DESCRIPTION
**Pull request by richsalz**
A handful of copyright nits.
Perhaps the only controversial one is the commit that removes Itanium assembler for RC4+MD5.  Although writing that sentence, I find it hard to believe.  @dot-asm are you okay?

### CODE REVIEW

File: apps/vms_term_sock.c [hunk 1]
```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright 2016 The OpenSSL Project Authors. All Rights Reserved.
```
Comment (levitte) [inline apps/vms_term_sock.c]: Has VSI granted us copyright?  If not, this is wrong.

File: apps/vms_term_sock.h [hunk 2]
```diff
@@ -1,4 +1,5 @@
 /*
+ * Copyright 2016 The OpenSSL Project Authors. All Rights Reserved.
```
Comment (levitte) [inline apps/vms_term_sock.h]: Has VSI granted us copyright?  If not, this is wrong.

**Comment by richsalz** [comment]
No, it is not wrong.  We have the copyright on what we did.  They have their copyright, too.

**Comment by levitte** [comment]
Are you going to tell me that source code reformatting is copyrightable? ;-)

**Comment by richsalz** [comment]
Every file in our distribution should have CLA coverage and our copyright.  It  can have others also.

**Comment by levitte** [comment]
I'm taking this to OMC mail...

**Comment by richsalz** [comment]
bump.

**Comment by richsalz** [comment]
ping @levitte having had the discussion, are you okay with the copyright on our changes in the VMS files now?

**Comment by richsalz** [comment]
squashed and merged, thanks.  We just reached an important milestone.

**Pull request closed without merging**